### ISSUE TITLE
Use Github Releases and integrate with open audio

### ISSUE THREAD

**Issue by RustoMCSpit** [dsp56300/gearmulator#226]
hi! this website says these plugins come in linux and clap https://linuxdaw.org/?d=Misc&f=clap&p=f&l=o

id like to add this here https://github.com/RustoMCSpit/awesome-linux-clap-list but i need the plugins to be added to the releases

**Comment by dsp56300** [comment]
You can download Linux clap versions from our website, Osirus as an example: https://dsp56300.wordpress.com/osirus/

**Comment by RustoMCSpit** [comment]
> You can download Linux clap versions from our website, Osirus as an example: https://dsp56300.wordpress.com/osirus/

do you have any plans to put these on github releases

**Comment by dsp56300** [comment]
Ah, that is what you mean. Yeah, github releases are planned

**Comment by RustoMCSpit** [comment]
off topic but close your milestones

**Comment by dsp56300** [comment]
See https://github.com/dsp56300/gearmulator/releases, Xenia will be added with the next release, but Osirus, OsTIrus and Vavra are already availabel

**Comment by RustoMCSpit** [comment]
this doesnt comply with open audio standards sadly so i cant  add it yet! i will contact the dev to come here

**Comment by RustoMCSpit** [comment]
also, what is this pop up about needing some additional files about? are they not included in the release? the way open audio works is that it uses github releases to access the files

**Comment by RustoMCSpit** [comment]
![image](https://github.com/user-attachments/assets/70b13f28-a97b-46fa-a633-ad5c9c7fd889)

**Comment by dddux** [comment]
You need ROM files - firmware to run the synths and they cannot be distributed with the releases. If you have hardware synths, you can dump them into a .mid or .syx files, or you can try finding them on the Internet. Cheers!

**Comment by dsp56300** [comment]
> oh i see, so i need to buy the synth first? as you have all the GUI set up, is there any discussion on ever replacing it with custom DSP

No, you may not be aware what this project is all about: We emulate synthesizers from the 90s. We provide the emulation of the hardware, but the firmware/rom/OS needs to be provided by the user. For very obvious reasons, we cannot and will not provide the firmware files as thexy are copyrighted.

What you're asking for here is to release an Nintendo 64 emulator on Steam and you want to include the games made by Nintendo. That is an obvious no go aswell.

Creating custom DSP code is not our intention, we want to preserve hardware synthesizers, providing the bit-exact output of the original devices.
